tests/functional/example/orion_config.yaml

```diff
@@ -0,0 +1,12 @@
+name: scikit-iris-tutorial
+
+max_trials: 1
+
+algorithms:
+  random:
+    seed: 1
+
+database:
```

Comment:
We should probably use PickledDB for the example.

Reply:
Agreed, it removes one point of friction for the learner.

Reply:
Agreed, it removes friction points for the learner.

Reply:
Actually, this test is not for the learner to see, it's for the CI system. All the other functional tests are using MongoDB.

Reply:
Oh, I did not realize this was in tests/. Still, the tests should run on the PickledDB as we are migrating to it as the default backend. For the tutorial I leave it as is then, to avoid adding documentation of the DB configuration in it. Once we migrate to PickledDB the tutorial will implicitly as well.

Reply:
Agreed.

Reply:
So we leave it as mongodb?

Reply:
Yes

Reply:
Why not using PickledDB if we plan to migrate to it by default?

Reply:
Because we havn't migrated it yet and do not know when we'll do it. The card in Trello has been created last year in June and was switched back and forth from the backlog and current sprint in October.

Changing just this test to PickleDB worsen the cohesiveness of the codebase to gain a potential minor time later. However, since there is no scheduled for this migration, there is a serious chance that we will end up with a fragmented test suite where one part will use MongoDB and the other PickleDB, rising the chance of not covering the right features and making it at the end more complicated overall to migrate.

Reply:
I agree about the fragmentation of the test suite, but a single migration would be overwhelming so we have no choice but to break it down and do it piece by piece. It has already began in many parts, so the fragmentation is already present. At least adding new tests with PickledDB would reduce the amount of remaining tests to migrate.

Reply:
I wasn't aware of the existing fragmentation when I looked for tests to get inspiration from. Then, it's settled: we have a clear stance already defined. I'll migrate the tests to PickleDB.
Thanks for the precisions!

Reply:
No problem, you raised good points!

Reply:
Implemented in https://github.com/Epistimio/orion/pull/339/commits/25067a9295b287da22237d88752fe08ed7294d2a